docs/process/process_areas/verification/guidance/verification_methods.rst

```diff
@@ -0,0 +1,348 @@
+   - **Invalid Input Classes**
+
+      Classes that contain invalid input values (out of range, incorrect data type, etc.).
+   - **Special Cases**
+
+      Classes representing specific or unusual input values that might trigger special behavior
+      in the software.
+
+   **3. Define Test Values**
+
+   Choose one representative test value from each equivalence class. It doesn't matter *which*
+   value you choose, as long as it's within the class. As an easy example consider the motor speed.
+   Test Values could be:
+
+   - Class 1 (Valid): 50 (A value within the valid range)
+   - Class 2 (Below Minimum): -10
+   - Class 3 (Above Maximum): 150
+
+   **4. Design Test Cases**
+
+   Create test cases using the representative values from each equivalence class.
+   Aim to cover all classes in your test suite.
+
+.. _ver_fuzzy:
+
+Fuzzy Testing
+"""""""""""""
+
+Fuzzy Testing, often called Fuzz Testing, is a software testing technique that involves providing
+invalid, unexpected, or random data as inputs to a component or system to detect defects. The goal
+of fuzzy testing is to expose unhandled exceptions, memory leaks, assertion failures, and other
+critical issues by generating semi-random inputs and observing the system's behavior.
+
+The following perspectives are included in ``fuzz-testing``:
+
+   **Detect Security Vulnerabilities**
+
+   By feeding malformed or unexpected inputs, fuzzy testing can reveal security vulnerabilities
+   like buffer overflows, SQL injections, and other input validation issues.
+
+   **Uncover Edge Cases**
+
+   Fuzzy testing can discover edge cases that might not be covered by traditional testing methods,
+   as it generates inputs that developers might not have anticipated.
+
+   **Automate Input Generation**
+
+   Instead of manually creating test inputs, fuzzy testing automates the process, allowing for a
+   much larger number of test cases to be executed.
+
+   **Improve Robustness**
+
+   By exposing the system to a wide variety of inputs, fuzzy testing helps ensure that the
+   software can handle unexpected situations gracefully.
+
+How to perform fuzzy testing:
+
+   **1. Identify Target Functions** Determine which functions or interfaces are suitable for
+   fuzzy testing, focusing on those that process external inputs.
+
+   **2. Define Input Structure** Understand the basic structure or format of valid inputs to
+   create effective fuzzers that can produce semi-valid inputs.
+
+   **3. Generate Fuzzy Inputs** Create inputs that are slightly malformed or unexpected, either
+   through complete randomization or by mutating valid inputs.
+
+   **4. Monitor System Behavior** Execute the system with fuzzy inputs and monitor for crashes,
+   hangs, memory leaks, or other unexpected behaviors.
+
+   **5. Analyze and Fix Issues** When a problem is detected, analyze the input that caused it
+   and fix the underlying issue in the code.
```

Comment:
@masc2023  this is quite high level so far. Do you see more to be added in the beginning?